src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -81,6 +84,23 @@ public void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
         indicateAddressBookChanged();
     }
 
+    /**
+     * Method is implemented in Model interface
+     * Functionality of method is exposed only in ModelManager
+     * Loops through Address Book to update the tags
+     * Address Book is then updated with updated tags of person
+     */
+    @Override
+    public void removeTag(Tag tags) throws DuplicatePersonException, PersonNotFoundException {
+        for (ReadOnlyPerson target: addressBook.getPersonList()) {
+            Person editedPerson = new Person(target);
+            Set<Tag> updatedTags = editedPerson.getTags();
+            updatedTags.remove(tags);
```

Comment:
See the comment here https://github.com/se-edu/addressbook-level4/pull/591/files

Tags from ObservableList cannot be removed successfully.